pypsa/linopt.py

```diff
@@ -234,6 +234,22 @@ def write_binary(n, axes):
     n.binaries_f.write(join_exprs('x' + _str_array(variables, True) + '\n'))
     return to_pandas(variables, *axes)
 
+
+def write_objective(n, terms):
+    """
+    Writer function for writing out one or mutliple objective terms.
```

Comment:
What does "one or multiple objective terms" mean?
```suggestion
    Writer function for writing out one or multiple objective terms.
```

Reply:
&rarr; one term or multiple terms